com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/utils/MigrationCfgUtils.java

```diff
@@ -889,8 +889,16 @@ protected VerifyResultMessages createVerifyResult(
 
     /** @return true if user should change the target character type's size to avoid data lost. */
     public boolean doesNeedToChangeCharacterTypeSize() {
-        return DatabaseType.MYSQL.equals(config.getSourceDBType())
-                || DatabaseType.ORACLE.equals(config.getSourceDBType());
+        boolean isEffectedByCharacterTypeSize;
+
+        if (config.targetIsFile()) {
+            isEffectedByCharacterTypeSize = !config.isAddUserSchema();
+        } else {
+            isEffectedByCharacterTypeSize = Integer.parseInt(config.getTargetDBVersion()) < 93;
```

Comment:
How about making the integer '93' a constant and using it?
ex) `CUBRID_9_3_UNDER`

Reply:
i think that is good suggestion and i agree that. but i think if we make '93' to static string variable, either '112' will apply that change. and this change need working in another issue.

Reply:
I don't understand that 112 is affected by the correction.
In the view of other developers, the integer '93' is thought to be misleading, so the opinion is to explain it with the variable name.

Reply:
i'm checking code now, some files use '112' as it is, and other some file use static variable. they are not consistent. 
so how about this? in this issue, make '93' to constant variable. and the 93 value which is not variables will change to constant variable in other issue.

Reply:
In the current issue, we generate a constant for '93', and we solve the constant problem of '112' by creating another issue.